apps/experiments/views/prompt.py

```diff
@@ -54,7 +55,18 @@ def get_queryset(self):
         return query_set
 
 
-class CreatePrompt(CreateView):
+class PromptViewMixin:
+    def form_valid(self, form):
+        if "{input}" in form.data["prompt"]:
```

Comment:
What about using the langchain PromptTemplate class to get the input variables. Seems like it would be more robust.

```
input_variables = PromptTemplate.from_template(form.data["prompt"]).input_variables
if input_variables - allowed_variables:
    ...
```
```

Reply:
https://github.com/dimagi/open-chat-studio/pull/149/commits/19c753af562df0b16feebb61bcd75479df3b30dd

![image](https://github.com/dimagi/open-chat-studio/assets/64970009/22e5e589-5245-440a-99e4-6671ded5e688)